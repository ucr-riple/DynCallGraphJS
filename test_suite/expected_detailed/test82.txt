{
    'global (test82.js:1:1:17:5)': [
        'anon (test82.js:1:2:17:2)'
    ],
    'anon (test82.js:16:3:16:19)': [
        'performanceNow (test82.js:4:22:8:6)'
    ]
}