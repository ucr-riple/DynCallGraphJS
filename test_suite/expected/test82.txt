{ '(test82.js:1:1:17:5)':
   [ '(test82.js:1:2:17:2)' ],
  '(test82.js:16:3:16:19)':
   [ '(test82.js:4:22:8:6)' ] }